### ISSUE TITLE
Console output not being announced by screen reader

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6944]
### System details

    RStudio Edition : Server
    RStudio Version : 1.4.376
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- turn on screen reader (tested JAWS, NVDA, VoiceOver)
- load RStudio Server, ensure screen-reader support is enabled
- <kbd>Ctrl+2</kbd> to ensure in Console, enter a command that produces output such as: `summary(airquality$Ozone)`

### Describe the problem in detail
- command output is not announced

### Describe the behavior you expected
- should have been announced as it was in RStudio 1.3

Other announcements such as "console cleared" after <kbd>Ctrl+L</kbd> are still announced, so it seems specific to console output scenario.

**Comment by gtritchie** [comment]
Repros locally when I build with `ant draft` but not via `ant devmode`. How nice.

**Comment by kfeinauer** [comment]
Verified fixed in RSP 1.4.953-1
